lib/saluki-event/src/lib.rs

```diff
@@ -25,11 +25,11 @@ pub enum DataType {
     /// Metrics.
     Metric,
 
-    /// Logs.
-    Log,
+    /// Datadog Events.
```

Comment:
In general, Saluki wants to be as agnostic as it can be: we're providing building blocks to making something like the Agent but we don't want to _necessarily_ write as if it's just the agent... so we'll keep names/descriptions generic where possible, and then specialize (like calling these "Datadog Events") where needed to disambiguate.